### ISSUE TITLE
Stuck on white screen!

### ISSUE THREAD

**Issue by Busybody64** [TeamREPENTOGON/REPENTOGON#409]
![image](https://github.com/TeamREPENTOGON/REPENTOGON/assets/92069026/3fe1bfab-bdee-4aaa-8fe0-acdad401986e)

I'm not sure what the fuck is happening, but the game crashes on a white screen on startup, and it doesn't go past it! I mean, i have almsot 200 mods, but...Even with all of them disabled, it STILL doesn't boot!

**Comment by Busybody64** [comment]
I found the problem! There's something wrong with the update checker! Whenever i disable it checking for updates, it works!

**Comment by Busybody64** [comment]
Maybe it's because I actually have a different window size?

**Comment by jsgnextortex** [comment]
No, it's an issue on some systems with folder permissions + updater, thats why disabling it on the dsound.ini fixes it.
